packages/runtime/container-runtime/src/blobManager/blobManagerSnapSum.ts

```diff
@@ -0,0 +1,133 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import {
+	AttachState,
+	type IContainerContext,
+} from "@fluidframework/container-definitions/internal";
+import { assert } from "@fluidframework/core-utils/internal";
+import { readAndParse } from "@fluidframework/driver-utils/internal";
+import type { ISummaryTreeWithStats } from "@fluidframework/runtime-definitions/internal";
+import { SummaryTreeBuilder } from "@fluidframework/runtime-utils/internal";
+import type { ITelemetryLoggerExt } from "@fluidframework/telemetry-utils/internal";
+
+/**
+ * Information from a snapshot needed to load BlobManager
+ * @legacy
+ * @alpha
+ */
+export interface IBlobManagerLoadInfo {
+	ids?: string[];
+	redirectTable?: [string, string][];
+}
+
+export const redirectTableBlobName = ".redirectTable";
+
+/**
+ * @internal
+ */
+export const blobsTreeName = ".blobs";
+
+/**
+ * Reads blobs needed to load BlobManager from storage.
+ *
+ */
+export const loadBlobManagerLoadInfo = async (
```

Comment:
Can't you just do `export const loadBlobManagerLoadInfo = loadV1`?

Reply:
And naming nit suggestion: `readBlobManagerLoadInfo` instead of `loadBlobManagerLoadInfo`? To avoid saying load twice - "load load info"